app/components/article-content.js

```diff
@@ -457,7 +457,7 @@ export default Component.extend(
             element.classList.add(shortClass);
             element.insertAdjacentHTML('beforeend',
               '<tr class=infobox-expand><td colspan=2><svg viewBox="0 0 12 7" class="icon">'
-       + '<use xlink:href="#chevron"></use></svg></td></tr>');
+              + '<use xlink:href="#chevron"></use></svg></td></tr>');
```

Comment:
What is this change ?

Reply:
Looks like my VSCode added that extra space.